Sources/UIExtensions/Extensions/View+Extensions.swift

```diff
@@ -20,6 +20,11 @@ extension View {
     public func squared(length: CGFloat, alignment: Alignment = .center) -> some View {
         frame(width: length, height: length, alignment: alignment)
     }
+
+    public func dump<T>(_ t: T) -> Self {
+        Swift.dump(t)
```

Comment:
This in a public library would ideally be behind a #if DEBUG condition and t being a autoclosure () -> T so it doesn't get evaluated in release.

But that's not super important.

Reply:
I would actually disagree. It's an edge case for this func, but especially in a public lib having something only available in DEBUG, and the consumer not being able to change it could be annoying. I feel it's appropriate to let the consumer decide if they want to use it in DEBUG only or RELEASE as well.